packer.json.dev

```diff
@@ -0,0 +1,48 @@
+{
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "name": "empire-us-east-1",
+      "access_key": "",
+      "secret_key": "",
```

Comment:
It'd be nice to parametrize the AWS credentials...most people should have these two already set when they're doing AWS work:

```
    "variables": {
        "aws_access_key": "{{ env `AWS_ACCESS_KEY_ID` }}" 
        , "aws_secret_key": "{{ env `AWS_SECRET_ACCESS_KEY` }}" 
    } 
    , "builders": [
        {   
            "type": "amazon-ebs"
            , "access_key": "{{user `aws_access_key`}}"
            , "secret_key": "{{user `aws_secret_key`}}"
        }
    ]
````